### PR TITLE
Handle the situation when our desired sandbox is empty

### PR DESCRIPTION
**Pull request by llelf**

### CODE REVIEW

**Comment by sol** [comment]
Neat! One more thing I can cross of the TODO list! :)

Can you add a test for this change to `test/Tinc/PackageSpec.hs`?

And congrats, you are probably one of the first `tinc` users outside of Zalora!

BTW, @soenkehahn and I have been working hard on support for "git dependencies".  That will hit master soon ;)

**Comment by llelf** [comment]
@sol today tinc has spoilt me, I don’t even want to work on a cabal idea I had :P

**Comment by sol** [comment]
I applied this and did some refactoring (see notes bellow).

File: test/Tinc/PackageSpec.hs [hunk 1]
```diff
@@ -18,6 +18,11 @@ spec = do
         , Package "getopt-generics" "0.6.3"
         ]
 
+  describe "parseEmptyInstallPlan" $ do
```
Comment (sol) [inline test/Tinc/PackageSpec.hs]: This should be a `context` of `parseInstallPlan` instead (see https://github.com/zalora/tinc/commit/ff08c48de3d354a0d4a1b20faa0c83ecd51a07a6).

File: src/Tinc/Package.hs [hunk 2]
```diff
@@ -18,7 +19,13 @@ parsePackage s = case break (== '-') (reverse s) of
   _ -> Package s ""
 
 parseInstallPlan :: String -> [Package]
-parseInstallPlan = map parsePackage . concatMap (take 1 . words) . drop 2 . lines
+parseInstallPlan (lines -> ("Resolving dependencies..."
+                            : what
+                            : maybePackages))
```
Comment (sol) [inline src/Tinc/Package.hs]: I avoid to align code in this way.  If we later rename `parseInstallPlan`, everything needs to be realigned.  In this specific case I also think that pattern guards are a valid option (see https://github.com/zalora/tinc/commit/86171ed6a49f4bbcc2c6cc0f76606be4396884e1).

**Comment by sol** [comment]
@llelf thanks a lot for tracking this down :)

**Comment by sol** [comment]
In addition, we print an error message on unexpected output from `cabal install --dry-run` now https://github.com/zalora/tinc/commit/aec54c1b1639dcf280c156291d8e2b20bdc3f850

**Pull request merged**